File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio6  tiempo de caida get/archives/6.py
import sys
import math

# Obtener los argumentos de la línea de comandos
num1 = int(sys.argv[1])
if (num1 > 0):
    mensaje = f"El tiempo de caida es {round(math.sqrt((2 * num1) / 9.81),2)}m/s^2"
elif (num1 == 0):
    mensaje = "El tiempo de caida es 0"
elif (num1 < 0):
    mensaje = "La altura no puede ser negativa"
else:
    mensaje = "ingrese un valor"

print(f"{mensaje}")

File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio6  tiempo de caida get/archives/6.js
function sumarJS() {
  const num1 = parseInt(document.getElementById("num1").value);
  if (isNaN(num1)) {
    mensaje = "Ingrese un número válido";
  }
  else if (num1 > 0) {
    mensaje = `El tiempo de caida es ${(Math.sqrt((2 * num1) / 9.81)).toFixed(2)} m/s²`;
  } else if (num1 === 0) {
    mensaje = "El tiempo de caida es 0";
  } else if (num1 < 0) {
    mensaje = "La altura no puede ser negativa";
  } else {
    mensaje = "ingrese un valor";
  }

  const resultadoDiv = document.getElementById("resultadoJavascript");
  resultadoDiv.innerHTML = `${(mensaje)}`;
}

